compiler/src/main/scala/org/scalajs/nscplugin/GenJSCode.scala

```diff
@@ -1201,20 +1184,9 @@ abstract class GenJSCode[G <: Global with Singleton](val global: G)
       }
 
       /* See BCodeHelprs.addForwarders in 2.12+ for why we normally use
```

Comment:
```suggestion
      /* See BCodeHelprs.addForwarders in 2.12+ for why we use
```
(then fold `exitingUncurry.` at the end of this line and make it a one-line comment instead, I guess)